guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,673 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9a65c44a",
+   "metadata": {},
+   "source": [
+    "# Introduction to ArcGIS Hub - Basic features\n",
+    "\n",
+    "ArcGIS Hub is a cloud-based engagement platform that enables organizations to communicate more effectively with their communities. ArcGIS Hub Basic is available with all ArcGIS Online subscriptions. Using Hub Basic, you can create unlimited websites and pages using the site editor and build a catalog of searchable content for each of your sites using the site's content library. Click [here](https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources) to learn more about ArcGIS Hub."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9cc8498c",
+   "metadata": {},
+   "source": [
+    "Note: The pattern to add, search, get, update, delete `Sites` and `Pages` is identical to that of __Initiatives__ (insert link to premium nb) as demonstrated here. \n",
+    "\n",
```

Comment:
* Add the word `and` before `delete`
* Is the plan to insert a hyperlink to another notebook instead of the test _(insert link to premium nb)_?n Is the plan to link it to the _guide-for-hub-premium.ipynb_ notebook once that is approved and merged?